syncoid

```diff
@@ -3,6 +3,9 @@
 # this software is licensed for use under the Free Software Foundation's GPL v3.0 license, as retrieved
 # from http://www.gnu.org/licenses/gpl-3.0.html on 2014-11-17.  A copy should also be available in this
 # project's Git repository at https://github.com/jimsalterjrs/sanoid/blob/master/LICENSE.
+# 
+# 2021-03-03: redbike
+# support for zfs holds added
```

Comment:
please remove this here, you will be credited in the changelog after the next release which includes the patch